doc/man1/openssl-s_server.pod

```diff
@@ -541,13 +541,12 @@ further information).
 
 =item B<-ssl2>, B<-ssl3>, B<-tls1>, B<-tls1_1>, B<-tls1_2>, B<-tls1_3>, B<-no_ssl2>, B<-no_ssl3>, B<-no_tls1>, B<-no_tls1_1>, B<-no_tls1_2>, B<-no_tls1_3>
 
-These options require or disable the use of the specified SSL or TLS protocols.
-By default, this command will negotiate the highest mutually supported
-protocol version.
-When a specific TLS version is required, only that version will be accepted
-from the client.
-Note that not all protocols and flags may be available, depending on how
-OpenSSL was built.
+See L<openssl(1)/TLS Version Options>.
+
+=item B<-dtls>, B<-dtls1>, B<-dtls1_2>
+
+These options make the command use DTLS protocols instead of TLS.
```

Comment:
make ... to use? Also this sentence is not present in the other manpages, should it be there?

Reply:
rewrote it:
```
These specify the use of DTLS protocols instead of TLS.
```
And added it to s_client.pod.